test-data/unit/semanal-errors.test

```diff
@@ -1060,6 +1060,19 @@ S = TypeVar('S', covariant=True, contravariant=True) \
     # E: TypeVar cannot be both covariant and contravariant
 [builtins fixtures/bool.pyi]
 
+[case testInvalidTypevarArgumentsGenericConstraint]
```

Comment:
Can you also add a test case that uses the new type parameter syntax (in `test-data/unit/check-python312.test`)?

Reply:
Thanks, I missed the fact that PEP 695 type vars are processed in a different part. Added some handling and tests